main.go

```diff
@@ -18,6 +18,7 @@ var (
 	flashbladeFlag = kingpin.Arg("flashblade", "Address of the target Flashblade.").Required().String()
 	portFlag       = kingpin.Flag("port", "Port to listen on.").Short('p').Default("9130").String()
 	fsMetricFlag   = kingpin.Flag("filesystem-metrics", "Export filesystem and usage data metrics for each user and group.").Default("false").Bool()
+	fsLimitFlag    = kingpin.Flag("filesystems", "Regexp limiting the filesystems for which metrics are exported").Default(".*").String()
```

Comment:
The flag name is `fsLimitFlag`, but ultimately its purpose is to drive the `FilterFilesystems` function. How about calling it `fsFilterFlag` instead?